modules/gis/docs/api-reference/geojson-to-binary.md

```diff
@@ -19,51 +19,92 @@ const geoJSONfeatures = await load(url, MVTLoader, loaderOptions);
 * Default options are:
 *
 * {
+*   coordLength: derived from data
+*   numericPropKeys: derived from data
 *   PositionDataType: Float32Array
 * }
 */
+const options = {PositionDataType: Float32Array};
 const binaryArrays = geojsonToBinary(geoJSONfeatures, options);
 ```
 
 ## Outputs
 
-### TypedArrays
-
 `geojsonToBinary` returns an object containing typed arrays sorted by geometry
-type. `positions` corresponds to 2D or 3D coordinates; `objectIds` returns the
-index of the vertex in the initial `features` array.
+type. `positions` is a flat array of coordinates; `globalFeatureIds` references
```

Comment:
For easier reading and a cleaner look, use a markdown table, or at least a bulleted list?